include/interface/libs/IMUKit.h

```diff
@@ -0,0 +1,32 @@
+// Leka - LekaOS
+// Copyright 2023 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <functional>
+namespace leka::interface {
+
+struct EulerAngles {
+	float pitch;
+	float roll;
+	float yaw;
+};
```

Comment:
can you move it out of `interface` so that we don't need to change code using them.

They could be forward declared here actually, to keep the definition in IMUKit.hpp